04-onion-routing.md

```diff
@@ -1201,6 +1298,7 @@ The following is an in-depth trace of an example of error message creation:
 [fips198]: http://csrc.nist.gov/publications/fips/fips198-1/FIPS-198-1_final.pdf
 [sec2]: http://www.secg.org/sec2-v2.pdf
 [rfc8439]: https://tools.ietf.org/html/rfc8439
+[routeblinding]: https://github.com/lightningnetwork/lightning-rfc/blob/route-blinding/proposals/route-blinding.md
```

Comment:
This needs to be in the repo if we want to reference it, presumably in this PR. I have a bunch of comments for this file to make clearer what's going on, but can't easily leave comments, and we shouldn't be referencing another branch of the same repo.